nautobot/core/templatetags/buttons.py

```diff
@@ -318,49 +317,55 @@ def consolidate_detail_view_action_buttons(context):
             </a>
         </button>
     """
-
+    dropdown_button_classes = "btn btn-warning rounded-end"
+    dropdown_button_style = (
+        "border-left: none; width: 5px; display: flex; justify-content: center; align-items: center;"
```

Comment:
What's all this needed for? Can we add it to the CSS class instead of explicit inline styling?

Reply:
Replaced all the styles with bootstrap classes except for "border-left: none; width: 5px;"